<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addPersonToTeamDiag</class>
 <widget class="QDialog" name="addPersonToTeamDiag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="firsnamelabel">
         <property name="text">
          <string>First name : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="firstnameLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="lastnameLabel">
         <property name="text">
          <string>Last name :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lastnameLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="nationalCodeLabel">
         <property name="text">
          <string>National Code : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nationalCodeLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="genderLabel">
         <property name="text">
          <string>Gender</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="genderComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="applyButton">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
